ppr-ui/src/components/tables/mhr/SearchedResults.vue

```diff
@@ -383,20 +428,52 @@ export default defineComponent({
     }
 
     const getOwnerStatus = (ownerStatus: string): string => {
-      if (ownerStatus) {
+      if (ownerStatus === 'PREVIOUS') {
         if (ownerStatus === 'PREVIOUS') return 'HISTORICAL'
         else return ownerStatus
       } else return ''
     }
 
+    // return adaptive text for owner status count(s)
+    const getOwnerStatusText = (item: ManufacturedHomeSearchResultIF): string => {
+      var returnText = ''
+      if (item.activeCount > 0) {
+        returnText += 'ACTIVE'
+        if (item.activeCount > 1) returnText += ' (' + item.activeCount + ')'
```

Comment:
There must be a better to re-write it. Maybe try string templating?

Reply:
An example: `returnText += \` (${item.activeCount})\``

Reply:
Just need to wrap your `${}` in back ticks

Reply:
Looks a lot better thanks